WearOAuth/oauth-device-grant/build.gradle

```diff
@@ -30,12 +45,14 @@ android {
     }
 }
 dependencies {
+    implementation project(':util')
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.5.0'
+    implementation 'androidx.core:core-ktx:1.6.0'
     implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1")
-    implementation("androidx.fragment:fragment-ktx:1.3.4")
+    implementation("androidx.fragment:fragment-ktx:1.3.5")
     implementation("androidx.wear:wear:1.1.0")
-    implementation("androidx.wear:wear-phone-interactions:1.0.0-alpha04")
     implementation 'com.google.android.gms:play-services-wearable:17.1.0'
+    implementation 'com.google.android.support:wearable:2.8.1'
+    compileOnly 'com.google.android.wearable:wearable:2.8.1'
```

Comment:
Is adding the support library required here? I think we are trying to remove it everywhere we can in the samples.

Reply:
Unfortunately, it is at this moment. It's used for the `RemoteIntent.startActivity()` method. This can be replaced once the corresponding method has been migrated to the androidx library.

Reply:
Ah, I see the use of `RemoteIntent`. The next version of `wear-remote-interactions` should have the replacement (I'm using a tip-of-tree snapshot version in a draft PR [here](https://github.com/android/wear-os-samples/pull/119))

I think it would make sense to either use the support for now or use the snapshot version. In both cases we should update to a normally released `androidx` version as soon as that happens.

Reply:
Ah sorry, I somehow missed your comment before mine!

The latest `androidx` release today should allow avoiding introducing the support library now!